Server/BackEndMonitoringServer/include/СContainerOfLogicalDisk.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "CLogicalDiskStatus.h"
+#include "�HardwareStatusSpecification.h"
```

Comment:
The same as above reduce the number of headers if possible and fix the mistype.

Reply:
Fixed, but wasn't able to move all includes(